pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -351,13 +351,15 @@ func TestPriorityQueue_AddUnschedulableIfNotPresent_Backoff(t *testing.T) {
 		q.AddUnschedulableIfNotPresent(unschedulablePod, oldCycle)
 	}
 
+	q.lock.RLock()
```

Comment:
It would be better to write a function(similar to [L109](https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/internal/queue/scheduling_queue_test.go#L109)) to wrap the lock and getting pod from `podBackoffQ`.

Reply:
In fact, I prefer to keep the test simple and verify the data like [L358](https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/internal/queue/scheduling_queue_test.go#L358).